auto_nag/scripts/workflow/no_severity.py

```diff
@@ -96,12 +95,39 @@ def set_people_to_nag(self, bug, buginfo):
     def get_bz_params(self, date):
         fields = ["triage_owner", "flags"]
         params = {
-            "bug_type": "defect",
             "include_fields": fields,
+            "email1": "intermittent-bug-filer@mozilla.bugs",
+            "emailreporter1": 1,
+            "emailtype1": "notequals",
+            "email2": "wptsync@mozilla.bugs",
+            "emailreporter2": 1,
+            "emailtype2": "notequals",
             "resolution": "---",
-            "f1": "priority",
+            "f1": "component",
             "o1": "equals",
-            "v1": "--",
+            "v1": "untriaged",
+            "f2": "OP",
+            "f3": "cf_status_firefox_nightly",
+            "o3": "equals",
+            "v3": "---",
+            "f4": "cf_status_firefox_beta",
+            "o4": "equals",
+            "v4": "---",
+            "f5": "cf_status_firefox_release",
+            "o5": "equals",
+            "v5": "---",
+            "f6": "cf_status_firefox_esr",
+            "o6": "equals",
+            "v6": "---",
+            "f7": "bug_type",
```

Comment:
is it on purpose to have the bug_type inside the parenthesis ?

Reply:
Yes. Again, part of the definition of untriaged. If a bug is a defect, it should have a status set in at least one of the four current trains. Those statuses are not applicable to those bug types in most circumstances.